pipeline/src/verify.rs

```diff
@@ -34,7 +34,6 @@ pub fn verify(temp_dir: &Path, name: &str, constants_name: Option<&str>) -> Resu
         return Err("Pil verifier run was unsuccessful.".to_string());
     } else {
         let output = String::from_utf8(verifier_output.stdout).unwrap();
-        log::error!("PIL verifier output: {}", output);
         if !output.trim().ends_with("PIL OK!!") {
             return Err("Verified did not say 'PIL OK' for {name}.".to_string());
```

Comment:
Shouldn't we log the output here?

Reply:
Ah ok, I thought the "did not say 'PIL OK'" was the main check. But I think it might still be useful to debug why it did not say ok if we have the output, right?